src/NuGet.Clients/PackageManagement.UI/Feeds/MultiSourcePackageMetadataProvider.cs

```diff
@@ -75,8 +90,34 @@ internal class MultiSourcePackageMetadataProvider : IPackageMetadataProvider
         public async Task<IPackageSearchMetadata> GetLatestPackageMetadataAsync(PackageIdentity identity,
             bool includePrerelease, CancellationToken cancellationToken)
         {
+            VersionRange allowedVersions = null;
```

Comment:
This should default to the *All* range instead of null.

``VersionRange allowedVersions = VersionRange.All;``